src/Microsoft.SqlTools.Hosting/Hosting/Contracts/Error.cs

```diff
@@ -20,5 +20,10 @@ public class Error
         /// Error message
         /// </summary>
         public string Message { get; set; }
+
+        public override string ToString()
+        {
+            return string.Format("Error(Code={0},Message='{1}'", Code, Message);
```

Comment:
>( [](start = 39, length = 1)

Missing )

Reply:
nit: suggest using string interpolation instead of string.Format, it is more modern and more readable as in:
return $"Error(Code={Code}, Message='{Message}'";

Reply:
Fixed both. Didn't know about string interpolation (show's I'm off my C# game in the last year or two) so thanks for recommending! We should consider updating across the codebase